lib/util.js

```diff
@@ -159,9 +159,9 @@ function stylizeNoColor(str, styleType) {
 
 
 function arrayToHash(array) {
-  var hash = {};
+  var hash = Object.create(null);;
 
-  array.forEach(function(val, idx) {
+  array.forEach(function(val) {
```

Comment:
Using `reduce()` is a bit misleading since nothing is actually being reduced, and I don't think turning it into a single expression is helpful in this case.

As for using arrow functions, personally only think they must be used to prevent `var self = this;` from happening, but also wouldn't be opposed if it were used. IOW, if it does change then cool, but it's not a blocker.